example/src/example.rs

```diff
@@ -12,3 +12,8 @@ fn main() {
         vprintf(b"%d %d %s %f\n\0".as_ptr() as *const c_char, v);
     }, 1, 2, b"salut!\0".as_ptr(), 32f32 as c_double);
 }
+
+#[cfg(target_os = "windows")]
+mod platform {
+	#[link(name = "legacy_stdio_definitions.lib")] extern {}
```

Comment:
Are you sure it is working? appveyor says it is failing https://ci.appveyor.com/project/GuillaumeGomez/va-list-rs/build/1.0.11/job/9e5fdf1r7npyx5od#L32